CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+## Contributing
+
+We appreciate your contributions to Hyperledger Fabric Private Chaincode.
+Any help is welcome and there is still much to do. 
+
+We adopt Fabric's developer guidelines where possible. Please visit Fabric's [contributors guide](http://hyperledger-fabric.readthedocs.io/en/latest/CONTRIBUTING.html) in the
+docs to learn more about it.
+
+Please go the [issues page](https://github.com/hyperledger-labs/fabric-private-chaincode/issues) to create a bug report
+or suggest a new feature.  Feel free to create a push request (PR) and link it to the corresponding issue.
+
+You can also reach us on [RocketChat](https://chat.hyperledger.org/) in #private-data-objects.
+
```

Comment:
I would add a reference to the Hyperledger Charter (https://www.hyperledger.org/about/charter),
and emphasize that 
* inbound contributions must be Apache2-compliant
* outbound contributions are Apache2

Reply:
ok, we can do that